_layouts/not_found.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
   <head>
-    {% include meta.html %} {% include styles.html %} {% include
-    google_analytics.html %}
+    {% include meta.html %}
+    {% include styles.html %}
+    {% include google_analytics.html %}
   </head>
   <body class="not-found">
     <div class="site-wrapper">{{ content }}</div>
   </body>
+  {% include dap.html %}
```

Comment:
I copied the style of `_layouts/main.html` here (DAP at bottom of page, GA at top)

The pageload numbers were off between DAP and GA (ex about 5 more views in GA than in DAP)....do we think that it's possible that if somebody closed a tab quickly that GA would see something but the browser would not have logged with DAP? Would it make sense to move the two script tags next to each other in the hopes that they start logging more similar results?

Reply:
> The pageload numbers were off between DAP and GA (ex about 5 more views in GA than in DAP)....do we think that it's possible that if somebody closed a tab quickly that GA would see something but the browser would not have logged with DAP? Would it make sense to move the two script tags next to each other in the hopes that they start logging more similar results?

Possibly, it may also depend when the scripts initialize (i.e. trigger immediately or on [`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)), or if one or the other might queue their logging (NewRelic's Browser SDK does this). Moving them closer together could help ensure the script code is executed at around the same time, though I kinda doubt this would make a huge difference.

Another plausible explanation is: Ad blockers. Most ad blockers will block Google Analytics scripts. I'm not sure they're always quite as aware of the DAP scripts.

Reply:
Do you think it's worth moving the tags together? Or just leaving as-is?

Reply:
> Do you think it's worth moving the tags together? Or just leaving as-is?

Seems reasonable, yeah. The base [`uswds-jekyll` template](https://github.com/18F/uswds-jekyll) (of which this repository seems to resemble) has [an analytics partial](https://github.com/18F/uswds-jekyll/blob/f322a81fabf887e41b989dcc59bf94ee8015a20c/_includes/analytics.html) where they print Google Analytics and DAP scripts together.

Reply:
Moved in d8dc4fa87c70a582263d20d86c249daaa0b16b26